website/docs/getting-started.mdx

```diff
@@ -80,97 +80,39 @@ The bootstrap command above does following:
 - Deploys Flux Components to your Kubernetes Cluster
 - Configures Flux components to track the path /clusters/my-cluster/ in the repository
 
-### Configure access to the dashboard
-
-For this guide we will use the cluster user, for complete documentation including how to configure an OIDC provider see the documentation [here](./configuration/securing-access-to-the-dashboard.mdx).
-
-We will generate a bcrypt hash for your chosen password and store it as a secret in Kubernetes. There are several different ways to generate a bcrypt hash, this guide uses `gitops get bcrypt-hash` from our CLI.
+### Install Weave GitOps
 
-1. Clone your git repository where Flux has been bootstrapped (you could skip this step if you performed previous steps in this doc).
+For this guide we will use the cluster user, for complete documentation including how to configure an OIDC provider see the documentation here.
```

Comment:
Ooooh, copy pasting back and forth has lost the link here

Reply:
Just merged it a sec ago, will add a tiny followup PR.